README.md

```diff
@@ -1,98 +1,29 @@
-# Vault.NET [![Build status](https://ci.appveyor.com/api/projects/status/784hg5j70vcnumeb/branch/master?svg=true)](https://ci.appveyor.com/project/chatham/vault-net/branch/master)
+# Vault.NET Local Certificate
```

Comment:
Please provide the description of the change in the description of the pull request.  This is the main README for the repository.